Storage/Rust/RustRemoteTabs.swift

```diff
@@ -0,0 +1,171 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/
+
+import Foundation
+import Shared
+@_exported import MozillaAppServices
+
+private let log = Logger.syncLogger
```

Comment:
We have a `Loggable` protocol to replace these `private let log....` danglies at the top of files. It'd be great if we could replace it with the new standard way of doing things. :)